include/znc/Message.h

```diff
@@ -121,6 +121,8 @@ class CMessage {
     void SetCommand(const CString& sCommand);
 
     const VCString& GetParams() const { return m_vsParams; }
+
+    VCString GetParamsSplit(unsigned int uIdx, unsigned int uLen = -1) const;
```

Comment:
Any comments? How is this different from any other function here?

Reply:
This returns a sub-vector formed from the parameters, allowing easy access to, for example, the 2nd parameter onwards.

Reply:
This allows inline use such as
```cpp
pChan->SetModes(Message.GetParam(2), Message.GetParamsSplit(3));
```
whereas removing the first parameter from a `GetParams()` call would require
```cpp
VCString vsParams = Message.GetParams();
vsParams.erase(vsParams.begin(), vsParams.begin()+3);
pChan->SetModes(Message.GetParam(2), vsParams);
```

Reply:
Could you add this to a comment? Answering my question here won't help anyone seeing the function in the header who has the same question.

Reply:
Done